lib/mail_handler_patch.rb

```diff
@@ -112,10 +113,17 @@ def receive_issue_reply_with_helpdesk(issue_id, from_journal=nil)
         return last_journal
       end
       
-      def custom_field_value(issue,name)
+      # differentiate between project field and issue field
+      def custom_field_value(issue,name,isproject_field = true)
```

Comment:
Since it's not used elsewhere it's over engineered, you're right. And what I've learned by writing some ruby, it's not nice ruby style, too ;).
I just did not want to redo the cleaning work that was done before as I like easy to read code and it was already in an own function.